src/app/payment/documentOnly/DocumentOnlyCustomForm.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { memo, FunctionComponent } from 'react';
+
+import { Fieldset } from '../../ui/form';
+import { CreditCardCustomField } from '../creditCard';
+
+export interface DocumentOnlyCustomFormProps {
+    additionalClassName: string;
+    autoComplete: string;
+    labelId: string;
+    name: string;
+}
+
+export interface DocumentOnlyCustomFormFieldsetValues {
```

Comment:
this doesnt seem to be used in here, is it the best place to define this interface? Maybe in the schema?